src/containers/Cech.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useRouteData } from 'react-static';
+import { Link } from '@reach/router';
+
+const CECH = () => {
```

Comment:
```suggestion
const CECHProject = () => {
```
Component names should be unique (or unique enough so as not to confuse them with other components)